cogboard-webapp/src/components/widgets/types/CheckboxWidget/index.js

```diff
@@ -21,14 +20,10 @@ const CheckboxWidget = props => {
   };
 
   return (
-    <WidgetIconButton aria-label="Toggle status" onClick={handleChangeStatus}>
-      <StatusIcon size="large" status={status}/>
+    <WidgetIconButton aria-label="Toggle status" data-cy={`checkbox-${status}`} onClick={handleChangeStatus}>
```

Comment:
do we really want `data-cy` to change with the status?

Reply:
Good point, I made it constant and added `aria-checked` attribute to be in sync with status